test/MLOps.NET.IntegrationTests/EndToEndTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using FluentAssertions;
+using Microsoft.ML;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MLOps.NET.IntegrationTests
+{
+    public class EndToEndTests : RepositoryTests
+    {
+        [TestMethod]
+        public async Task DeployModel_GivenACompleteRun_ShouldDeployModel()
```

Comment:
ℹ️ Dev note: I've added this class in case we want to more cross-cutting tests, e.g. start to finish that doesn't drill into specifics but ensures that our flow is intact